docs/pages/workflow/continuous-native-generation.mdx

```diff
@@ -25,91 +26,254 @@ Expo as a framework enables **Continuous Native Generation** by combining the fo
+    '$ npx expo run:android',
+    '',
+    '# Build your native iOS project',
+    '$ npx expo run:ios',
+  ]}
+/>
+
+If native directories are absent, `npx expo prebuild` will run once for the specific platform. On subsequent uses of these `run` commands, manually run `npx expo prebuild` to ensure the native code is freshly synchronized with your local configuration.
+
+## Platform support
+
+Prebuild currently supports Android and iOS. Web support is not required because there is no native project to generate for the web and the web app it runs in a web browser.Use the `--platform` option to build for individual platforms:
+
+<Terminal cmd={['$ npx expo prebuild --platform ios']} />
+
+## Dependencies
+
+Prebuild begins by initializing new native projects from a template corresponding to each Expo SDK version. This also aligns with specific React and React Native versions. Your project's React and React Native versions will update to the versions specified in the `dependencies` field of the [template's **package.json**](https://github.com/expo/expo/tree/main/templates/expo-template-bare-minimum) if they differ.
+
+You can skip changing npm package versions with the `--skip-dependency-update` option:
+
+<Terminal cmd={['$ npx expo prebuild --skip-dependency-update react-native,react']} />
+
+## Package managers
+
+When the [dependencies](#dependencies) are changed, Prebuild will reinstall libraries using the package manager that is currently used in the project (this is inferred from the lockfile). You can force a specific package manager by providing one of: `--npm`, `--yarn`, `--pnpm`.
+
+All installations can be skipped by passing the `--no-install` command, which is useful for testing generation quickly.
+
+## Clean
+
+The `--clean` option deletes any existing native directories before generating. Re-running `npx expo prebuild` without the `--clean` option will layer changes on top of the existing files, which is faster, but may not produce the same results in some cases.
+
+For example, some config plugins aren't idempotent. When a project utilizes multiple "dangerous modifiers" to add regex changes to an app's code, it can lead to unexpected behavior. This is why using the `--clean` option is the safest way to use the prebuild command and is generally recommended in most cases.
+
+#### Using `--clean` option
+
+When using the `--clean` option, you'll be warned to have a clean git status due its destructive nature. This prompt is optional and will be skipped when encountered in CI. You can disable this check by enabling the environment variable `EXPO_NO_GIT_STATUS=1`.
+
+This prompts aims to encourage developers to add **android** and **ios** directories to the project's **.gitignore** or **.easignore**. This ensures that these directories are always generated by prebuild. However, this can make custom config plugins harder to develop so we haven't introduced any mechanism to enforce this behavior.
```

Comment:
```suggestion
This prompts aims to encourage developers to add **android** and **ios** directories to the project's **.gitignore** or **.easignore**. This ensures that these directories are always generated by prebuild. However, this can make custom config plugins more difficult to develop, so we haven't introduced any mechanism to enforce this behavior.
```
I'm actually not sure why this would make config plugins more difficult to develop. I would expect to be continuously cleaning my native directories and diffing against some other set of native files, or maybe I would be on a branch where I committed the native files, and then I would keep cleaning until my diff looked right.

Reply:
I copied all prebuild info from the existing Prebuild page. I am not sure either but might be a good question to ask to a Subject Matter Expert.

Reply:
We could also consider striking the paragraph entirely. I'm not sure the prompt does a particularly good job of encouraging developers to add those folders to gitignore. The prompt itself doesn't say anything to recommend this.